The scripts are installed as 'data' rather than scripts.
http://code.google.com/p/python-gflags/issues/detail?id=12

From d544cdad63faa5a53ba524e39f091f4a18e1eeb8 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@gentoo.org>
Date: Sun, 16 Dec 2012 16:27:13 +0100
Subject: [PATCH] Install Python scripts using 'scripts' key.

This is necessary to ensure that the shebangs are properly replaced when
installing for multiple Python implementations.
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 573db2d..ddad318 100755
--- a/setup.py
+++ b/setup.py
@@ -39,6 +39,6 @@ setup(name='python-gflags',
       author_email='google-gflags@googlegroups.com',
       url='https://github.com/google/python-gflags',
       py_modules=['gflags'],
-      data_files=[('bin', ['gflags2man.py'])],
+      scripts=['gflags2man.py'],
       include_package_data=True,
       )
-- 
1.8.0.2

